WalletWasabi.Tests/UnitTests/BitcoinCore/MempoolTests.cs

```diff
@@ -18,31 +20,30 @@ public class MempoolTests
 	public void AddRemoveTransaction()
 	{
 		// Create the tx1 transaction.
-		Transaction tx1;
+
+		SmartTransaction tx1 = new SmartTransaction(Network.Main.CreateTransaction(), Height.Mempool);
 		OutPoint tx1prevOut1;
 		{
-			tx1 = Network.Main.CreateTransaction();
 			tx1prevOut1 = new(hashIn: new uint256(0x1), nIn: 0);
-			tx1.Inputs.Add(new TxIn(tx1prevOut1));
+			tx1.Transaction.Inputs.Add(new TxIn(tx1prevOut1));
 
 			OutPoint tx1prevOut2 = new(hashIn: new uint256(0x2), nIn: 0);
-			tx1.Inputs.Add(new TxIn(tx1prevOut2));
+			tx1.Transaction.Inputs.Add(new TxIn(tx1prevOut2));
 
 			TxOut tx1Out1 = new(Money.Coins(0.1m), scriptPubKey: Script.Empty);
-			tx1.Outputs.Add(tx1Out1);
+			tx1.Transaction.Outputs.Add(tx1Out1);
 		}
 
 		// Create the tx2 transaction that spends the same tx1prevOut1 as tx1.
-		Transaction tx2;
+		SmartTransaction tx2 = new SmartTransaction(Network.Main.CreateTransaction(), Height.Mempool);
```

Comment:
Instead of `SmartTransaction = `you can write `var`, because on the same line you make this object.